### PR TITLE
Add counter to next birthday and display age

### PR DESCRIPTION
**Pull request by arnimattr**
I implemented the change requested on [reddit](https://www.reddit.com/r/unixporn/comments/wrcwkz/comment/ikrrh62) and added a column that contains the age at the next birthday.
I'm sorry if my additions are not rust-typical code as I'm not used to the language, but I tried my best :)

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -79,8 +93,8 @@ fn main() -> io::Result<()> {
             }
             "add" => {
                 if args.len() < 4 {
-                    println!("Enter the required arguments! \"bday add [name] [date]\"");
-                    exit(1);
+                    println!("Enter the required arguments! \"bday add [name] [day-month-year]\"");
+                    exit(0);
```
Comment (arnimattr) [inline src/main.rs:97]: I changed it to exit normally, but you can change it back

File: src/main.rs [hunk 2]
```diff
@@ -33,7 +33,7 @@ impl Person {
 
 fn main() -> io::Result<()> {
     let mut bdays = Bdays::new();
-    let path: &String = &format!("/home/{}/.config/bdays.json", env!("USER"));
+    let path: &String = &format!("{}/.config/bdays.json", env!("HOME"));
```
Comment (arnimattr) [inline src/main.rs:36]: /home/user only works on linux

**Comment by rv178** [comment]
Amazing! Thank you for the PR :)

File: src/main.rs (hunk 2 above)
Comment (Shinyzenith) [reply src/main.rs:36]: Funny cuz the change you made is also linux only if I'm not mistaken

Comment (rv178) [reply src/main.rs:36]: I see, ty!

Comment (rv178) [reply src/main.rs:36]: > Funny cuz the change you made is also linux only if I'm not mistaken

wait yeah, / doesn't work on windows paths right?

Comment (rv178) [reply src/main.rs:36]: eh doesn't really matter, this is targeted towards linux users anyway :rofl:

**Pull request merged**